financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/FinancialConnectionsPlaygroundViewModel.kt

```diff
@@ -315,10 +318,18 @@ sealed class FinancialConnectionsPlaygroundViewEffect {
     ) : FinancialConnectionsPlaygroundViewEffect()
 }
 
-data class FinancialConnectionsPlaygroundState(
+internal data class FinancialConnectionsPlaygroundState(
     val backendUrl: String = "",
+    val settings: PlaygroundSettings,
     val loading: Boolean = false,
     val publishableKey: String? = null,
     val status: List<String> = emptyList(),
     val emittedEvents: List<String> = emptyList()
-)
+) {
+
+    constructor(application: Application) : this(
+        settings = PlaygroundSettings.createFromSharedPreferences(application)
+    )
+
+    val flow = settings[FlowDefinition]
```

Comment:
settings is mutable, should this be a getter instead?

Reply:
it's actually inmutable : )
<img width="432" alt="image" src="https://github.com/stripe/stripe-android/assets/99293320/4798d19b-cc1f-4c20-8ac8-fea7840258a4">